### PR TITLE
feat: example支持外链

### PR DESCRIPTION
**Pull request by FanSun521**

### CODE REVIEW

File: src/pages/Examples/components/GalleryPageContent/DemoCard/index.tsx [hunk 1]
```diff
@@ -17,31 +17,46 @@ export const DemoCard: React.FC<DemoCardProps> = (props) => {
   const locale = useLocale();
 
   const renderCardInternal = () => {
-    const img = demo.screenshot ||
+    const img =
+      demo.screenshot ||
       'https://gw.alipayobjects.com/os/s/prod/antv/assets/image/screenshot-placeholder-b8e70.png';
```
Comment (boyongjiong) [inline src/pages/Examples/components/GalleryPageContent/DemoCard/index.tsx]: [screenshot-placeholder-white](https://cdn.jsdelivr.net/gh/Logic-Flow/static@latest/docs/screenshot-placeholder-white.png)

替换成这个

File: src/pages/Examples/components/GalleryPageContent/DemoCard/index.tsx [hunk 2]
```diff
@@ -17,31 +17,46 @@ export const DemoCard: React.FC<DemoCardProps> = (props) => {
...
+            backgroundImage: `url("${img}")`,
+          }}
+        />
       </>
     );
   };
 
-  return (
+  return demo.isExternal ? (
+    <a className={styles.galleryCardLink} href={demo.previewUrl}>
```
Comment (boyongjiong) [inline src/pages/Examples/components/GalleryPageContent/DemoCard/index.tsx]: _blank

Comment (boyongjiong) [reply src/pages/Examples/components/GalleryPageContent/DemoCard/index.tsx]: https://umijs.org/docs/api/api#link

File: src/slots/ExampleSider/index.tsx [hunk 3]
```diff
@@ -14,7 +14,7 @@ import styles from './index.module.less';
 
 // menu icon
 const MenuIcon = createFromIconfontCN({
-  scriptUrl: '//at.alicdn.com/t/font_470089_1lnym745udm.js' // generated by iconfont.cn
+  scriptUrl: '//at.alicdn.com/t/c/font_4629554_4j922h52f0t.css' // generated by iconfont.cn
```
Comment (boyongjiong) [inline src/slots/ExampleSider/index.tsx:17]: 给我加下权限

File: src/pages/Examples/components/GalleryPageContent/DemoCard/index.tsx (hunk 2 above)
Comment (FanSun521) [reply src/pages/Examples/components/GalleryPageContent/DemoCard/index.tsx]: umi的Link 加了reloadDocument还是网址拼接

**Review by boyongjiong — APPROVED** [review]

**Pull request merged**